pysat/_files.py

```diff
@@ -103,36 +116,36 @@ class uses the first directory with relevant data, stored in data_path.
     new files and filtering out empty files.
 
     User should generally use the interface provided by a pysat.Instrument
-    instance. Exceptions are the classmethod from_os, provided to assist
+    instance. Exceptions are the classmethod `from_os`, provided to assist
     in generating the appropriate output for an instrument routine.
 
     Examples
     --------
     ::
 
-        # convenient file access
+        # Convenient file access.
         inst = pysat.Instrument(platform=platform, name=name, tag=tag,
                                 inst_id=inst_id)
-        # first file
+        # First file.
         inst.files[0]
 
-        # files from start up to stop (exclusive on stop)
+        # Files from start up to stop (exclusive on stop).
         start = dt.datetime(2009,1,1)
         stop = dt.datetime(2009,1,3)
         print(inst.files[start:stop])
 
-        # files for date
+        # Files for date.
         print(inst.files[start])
 
-        # files by slicing
+        # Files by slicing.
         print(inst.files[0:4])
 
-        # get a list of new files
-        # new files are those that weren't present the last time
-        # a given instrument's file list was stored
+        # Get a list of new files.
+        # New files are those that weren't present the last time
```

Comment:
Combine these two to not be on separate lines now that the period is acting like an end of sentence instead of newline serving that function.

Reply:
I looked up the numpydoc docstring format, or something like that, when I saw the posted standard was for comments to have periods. Thought I was behind more than trying to apply something new. Combined.

Reply:
It was PEP8, 
```
Comments should be complete sentences.
```

Of course it also says,
```
You should use two spaces after a sentence-ending period in multi- sentence comments
```
which I'm not doing. Feels old. Also, just saw it this time round.

Reply:
Our own contributing standards also specify that there should be a period here.  I wasn't objecting to that, just the unnecessary line break.